app/controllers/api/projects.php

```diff
@@ -90,7 +89,34 @@
         }
 
         $projectId = ($projectId == 'unique()') ? ID::unique() : $projectId;
+
+        $backups['database_db_fra1_02'] = ['from' => '7:30', 'to' => '8:15'];
+        $backups['database_db_fra1_03'] = ['from' => '10:30', 'to' => '11:15'];
+        $backups['database_db_fra1_04'] = ['from' => '13:30', 'to' => '14:15'];
+
         $databases = Config::getParam('pools-database', []);
+
+        /**
+         * Extract db from list while backing
+         */
+        if (count($databases) > 1) {
+            $now = new \DateTime();
+
+            foreach ($databases as $index => $database) {
+                if (empty($backups[$database])) {
+                    continue;
+                }
+                $backup = $backups[$database];
+                $from = DateTime::createFromFormat('H:i', $backup['from']);
+                $to = DateTime::createFromFormat('H:i', $backup['to']);
+                if ($now >= $from && $now <= $to) {
+                    unset($databases[$index]);
+                    break;
+                }
+            }
+            $databases = array_values($databases);
```

Comment:
```suggestion
```
This line seems unnecessary 🤔 

<img width="1388" alt="Screenshot 2023-06-06 at 5 51 05 PM" src="https://github.com/appwrite/appwrite/assets/20852629/55411341-e219-4667-aa37-acf56b65c7e2">